packages/scandipwa/src/component/MyAccountCustomerForm/MyAccountCustomerForm.container.js

```diff
@@ -73,7 +73,7 @@ export class MyAccountCustomerFormContainer extends PureComponent {
             handleChangeEmailCheckbox,
             handleChangePasswordCheckbox,
             currentPassword,
-            email: email || currentCustomerEmail
+            email: email === null ? currentCustomerEmail : email
```

Comment:
Like in first case if email null/undefined it will anyway move to second option currentCustomerEmail.
So what have changed ?

Reply:
because by making it specific only null, it will prevent the loop by accepting an empty string that only happens when the input is emptied by the user and make the current email only set in first render only